Description: Link with pthread to fix FTBFS: undefined reference to `pthread_create'
Author: Andreas Moog <andreas.moog@warperbbs.de>
Bug: http://synergy-foss.org/spit/issues/details/3924/

Index: synergy-1.5.0/src/lib/arch/CMakeLists.txt
===================================================================
--- synergy-1.5.0.orig/src/lib/arch/CMakeLists.txt	2014-05-25 00:13:19.824373253 +0200
+++ synergy-1.5.0/src/lib/arch/CMakeLists.txt	2014-05-25 00:13:28.280373181 +0200
@@ -49,6 +49,10 @@
 
 add_library(arch STATIC ${sources})
 
+if (HAVE_PTHREAD)
+	target_link_libraries (arch pthread)
+endif()
+
 if (UNIX)
 	target_link_libraries(arch ${libs})
 endif()
